src/ext/vendor_stdatomic.h

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
Since this is the second lib we've vendored in, we should probably start considering were we put 3rd-party code. I put the MessagePack files in `src/ext/mpack` but I'm not 100% happy with that location. And dropping in a `vendor` folder could get confusing with Composer. So I'm open to ideas, but I'd love to consolidate all the vendered code. :)

Also - could we append this line to the `.gitattributes` to have this auto-collapse in diffs:

```
src/ext/vendor_stdatomic.h*     linguist-generated=true
```

If we throw all the vendored code into one directory, we can just always have it auto-collapse every time we add/update:

```
src/ext/vendored_code_here/*     linguist-generated=true
```

Anyway - I'd love to hear your thoughts. :)

Reply:
Need to vendor in this file surprised me abit. Hopefully we'll be able to get rid of this once we'll be able to upgrade GCC in 7.3 build images 
`stdatomic.h` should be part of C11 standard but GCC lagged "a bit" in implementing it :(

Reply:
Basically I hope to get rid of `vendor_stdatomic.h` "soon" so the sorest it looks the higher chance we'll remember to remove it.

overall I don't mind having `vendor` folder. But I worry about being able to set include paths correctly so that the files don't need to be modified when they reference themselves under specifc paths.  "Practicality beats purity" etc :)